layouts/partials/deprecation-warning.html

```diff
@@ -9,4 +9,15 @@ <h3>
     </p>
   </div>
 </section>
+{{ else if and (eq .Section "blog") .Date (.Date.Before (now.AddDate -1 0 0)) -}}
+{{ $title := .Param "outdated_page.title" | default (T "outdated_page_title") -}}
+{{ $msg := .Param "outdated_page.message" | default (T "outdated_page_message") -}}
+{{ $reason := .Param "outdated_page.reason" | default (T "outdated_page_reason") -}}
+{{ $msg = replaceRE "REASON" $reason $msg -}}
+<section id="deprecation-warning">
```

Comment:
To make it easier to customize the style of the blog warning separately, I would:
```suggestion
<section id="deprecation-warning" class="blog-outdated-warning">
```